api/src/main/resources/notebooks.yaml

```diff
@@ -1375,14 +1375,6 @@ definitions:
       userJupyterExtensionConfig:
         $ref: "#/definitions/UserJupyterExtensionConfig"
         description: Jupyter extensions to be installed in the notebook
-      jupyterExtensionUri:
```

Comment:
Deprecated and unused by us

Reply:
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml#L1112